frontend/packages/knative-plugin/src/actions/add-event-sink.tsx

```diff
@@ -12,6 +12,7 @@ export const AddEventSinkAction = (
   application?: string,
   contextSource?: string,
   path?: string,
+  isEventSinkTypeEnabled?: boolean,
```

Comment:
We should not call this function if the `Action` is not enabled "for any reason"